geomstats/information_geometry/dirichlet.py

```diff
@@ -480,12 +483,143 @@ def exp(self, tangent_vec, base_point, n_steps=N_STEPS):
 
         return exp
 
-    def _geodesic_bvp(
+    def _approx_geodesic_bvp(
         self,
         initial_point,
         end_point,
-        n_steps=N_STEPS,
-        jacobian=False,
+        degree=5,
+        method="BFGS",
+        n_times=200,
+        jac_on=True,
+    ):
+        """Solve approximation of the geodesic boundary value problem.
```

Comment:
Do you have a reference for this?

Reply:
No not yet, but hopefully I will have one soon.